spec/src/modules/tracker.js

```diff
@@ -20,6 +20,7 @@ const { fetch } = fetchPonyfill({ Promise });
 
```

Comment:
The only changes are that we wrap expectations in a `setTimeout` of `waitInterval` to ensure the component has had time to actually make the request (since they are now deferred by a certain amount of time).